### DISCUSSION TITLE
❇️ New Project Logo ❇️

### DISCUSSION THREAD

**Discussion by mishmanners** [mishmanners/GitHub-Like-A-Boss#52] [Q&A] [answered]
Okay we're getting a new logo. We want it to showcase all the cool things we stand for. It needs to be bold and bright, but simplistic.

Who are some good designers we can reach out to?

**Comment by HUSK1E** [comment]
Do we need a new logo?

**Reply by mishmanners** [reply]
This is about _who_ we should choose to have as our designer. If you think this needs more Discussion about whether we should have a new logo, let's open a new Discussion.

**Comment by HackathonQueen** [comment, marked as answer]
I think we should go with Eva again for logo design. She was great to work with. I think we do need a new logo. We need to modernise our look.

**Comment by mishmanners** [comment]
A new logo is something we discussed in the team meeting last week. Many people in the team think we need to modernise our look. What do you all think?

**Comment by HUSK1E** [comment]
Yeah I do like Eva. Her work is amazing. But again, do we need a new logo?

**Comment by EsportsQueen** [comment]
Maybe we should have a bigger discussion around this? 🤷

**Comment by mishmanners** [comment]
You're right @EsportsQueen. Okay converting this to a Discussion.
